### PR TITLE
Notebook3 edits

### PR DESCRIPTION
**Pull request by SophieRowland**

### CODE REVIEW

File: docs/src/03-metadata-analysis.jmd [hunk 1]
```diff
@@ -37,6 +40,9 @@ histogram(sampleinfo[:nsamples], legend=false,
     xlabel="# of fecal samples", ylabel="# of subjects")
 ```
 
+**This graph includes fecal and ethanol samples, yes? Do we want to distinguish
+these? Because it's misleading how many _timepoints_ we have for each subject**
```
Comment (kescobo) [inline docs/src/03-metadata-analysis.jmd]: Yes - that's addressed further down.

File: docs/src/03-metadata-analysis.jmd [hunk 2]
```diff
@@ -71,10 +80,12 @@ highsamplers = @linq filter(r-> r[:studyID] in highsamplers[:studyID], allmeta)
 
 first(highsamplers, 15) |> randrows
 ```
+**Shows `missing` values `DOM` too. Though many of those are same timepoints
```
Comment (kescobo) [inline docs/src/03-metadata-analysis.jmd]: Good catch, fixed.

File: docs/src/03-metadata-analysis.jmd [hunk 3]
```diff
@@ -86,6 +97,12 @@ sort!(mgxsamples, :studyID);
 first(mgxsamples, 5) |> randrows
 ```
 
+**Is it worth noting that after a certain batch (will have to check exact #) we
```
Comment (kescobo) [inline docs/src/03-metadata-analysis.jmd]: Not sure exactly what you mean, but if you think it's worth clarifying go ahead and write it up here.

File: docs/src/03-metadata-analysis.jmd [hunk 4]
```diff
@@ -86,6 +97,12 @@ sort!(mgxsamples, :studyID);
...
+only sent the first CollectionRep of the genotek fecal samples?**
+
+**We should add a full breakdown of the `sampleID` naming conventions somewhere.
```
Comment (kescobo) [inline docs/src/03-metadata-analysis.jmd]: Other discussion in #7

**Comment by kescobo** [comment]
There were more modifications here than other notebooks, please take a look. If you have additional comments, you can use the line comment feature as I did above - let me know if you need help figuring that out.

**Comment by SophieRowland** [comment]
Notebook 3 has been reviewed

**Pull request merged**